workspaces/lightspeed/plugins/lightspeed-backend/README.md

```diff
@@ -35,7 +35,8 @@ lightspeed:
     - id: <server id>
       url: <serverURL>
       token: <api key>
-      questionValidation: true # Optional - To disable question (prompt) validation set it to false.
+  questionValidation: true # Optional - To disable question (prompt) validation set it to false.
+  port: <portNumber> # Optional - Change the LS service port nubmer. Defaults to 8080.
```

Comment:
Do we want to distinguish between RCS port 8080 and LS Backend port 7007 or Frontend 3000? Especially since it is `lightspeed.port`

Reply:
Updated it to `servicePort`.